packages/reassure-cli/src/commands/measure.ts

```diff
@@ -23,6 +22,7 @@ interface MeasureOptions extends CommonOptions {
 
 export async function run(options: MeasureOptions) {
   configureLoggerOptions(options);
+  hello(options);
```

Comment:
IMO we should not runt `hello` or `bye` by default for any other commend beside `init` as they will be annoying to the user.